vipps-ecom-api.md

```diff
@@ -214,6 +216,44 @@ To perform an express checkout, the merchant needs to send
 `"paymentType": "eComm Express Payment"` as part of initiate payment, and
 support the `shippingDetails` and `consent` endpoints.
 
+### Old and new express checkout flow
+
+This does require some background info - please bear with us.
+
+In the old flow for the express checkout it was easy for a user to overlook
+that the correct address and shipping method was used, as both were simply
+displayed on the payment confirmation page. The user did not have to actively
+choose and address and shipping method. This resulted in some users
+inadvertently confirming the wrong address and possibly also selecting the
+wrong shipping method. To fix this, we made it mandatory to select and address
+and shipping method. This was released as a minor update of the Vipps app.
+
+That worked well for all merchants that used Vipps Hurtigkasse _as intended_,
```

Comment:
Don't think we need to go into this detail
Just say we added an optional longer flow where merchants can choose if this is appropriate

Reply:
There are several thousand users out there, so _some_ background info is required. May shorten it a bit later.